package.json

```diff
@@ -6,18 +6,27 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "prepare": "./scripts/prepare",
+    "lint": "eslint ./src/ --fix",
+    "spell": "cspell ./src/**/*"
```

Comment:
It will not work for the direct child of `src`. We should use `./src/**.`

Reply:
it actually works for them too

Reply:
can you give an example path that it would not work on?

Reply:
./src/file.ts